frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -38,8 +38,9 @@ const Item: React.FC<ItemProps> = ({
         onClick={(e: React.SyntheticEvent) => navigateTo(e, resolvedUrl)}
         href={resolvedUrl}
         title={label}
-        iconImg={icon}
+        iconImg={typeof icon === 'string' ? icon : undefined}
         iconClass={iconClass}
+        icon={typeof icon === 'object' ? icon : undefined}
```

Comment:
```suggestion
        icon={React.isValidElement(icon) ? icon : undefined}
```
Pretty sure this is the check you want.

Reply:
Indeed i do!

Reply:
fixed. thanks @andrewballantyne